client/src/components/common/Notification/Notification.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <VSnackbar
+        v-model="notification.showing"
+        :color="notification.type"
+        :timeout="timeout"
+        :top="top"
+        :style="`top: ${index * 80 + 15}px`"
+    >
+        <VRow>
+            <VCol cols="12" md="11">
+                {{ notification.text }}
+            </VCol>
+            <VCol class="text-right" align-self="center" cols="12" md="1">
+                <span class="close" @click="closeSnackbar">
+                    <VIcon size="14">
+                        mdi-close
+                    </VIcon>
+                </span>
+            </VCol>
+        </VRow>
+    </VSnackbar>
+</template>
+
+<script>
+export default {
+    name: 'Notification',
+
+    props: {
```

Comment:
destruct notification on props you use in the component, because what the shape of notification are you expected here? nobody knows

Reply:
What do you mean? Remove props?

Reply:
nope, here you have object that can contain anything, it is better divide it on props

id: String
text: String
show: Boolean
etc.